spec/gobstones/expectations/has_procedure_usage_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rspec'
+
+require_relative '../../../lib/gobstones'
+
+include Gobstones::Expectations
+
+describe HasProcedureUsage do
+  context '#value' do
+    let(:runner) { ExpectationsRunner.new('python .heroku/vendor/pygobstones/language/vgbs/gbs.py', 'program { IrAlBorde(Oeste) }') }
+    let(:results) { runner.run! [HasProcedureUsage.new('IrAlBorde'), HasProcedureUsage.new('Mover')] }
+    it { expect(results.first[1]).to be_true }
```

Comment:
Agree, could you submit an issue?

El jue, may 28, 2015 19:52, Franco Leonardo Bulgarelli <
notifications@github.com> escribió:

> In spec/gobstones/expectations/has_procedure_usage_spec.rb
> <https://github.com/uqbar-project/gobstones-spec/pull/9#discussion_r31285900>
> :
>
> > @@ -0,0 +1,14 @@
> > +require 'rspec'
> > +
> > +require_relative '../../../lib/gobstones'
> > +
> > +include Gobstones::Expectations
> > +
> > +describe HasProcedureUsage do
> > +  context '#value' do
> > +    let(:runner) { ExpectationsRunner.new('python .heroku/vendor/pygobstones/language/vgbs/gbs.py', 'program { IrAlBorde(Oeste) }') }
> > +    let(:results) { runner.run! [HasProcedureUsage.new('IrAlBorde'), HasProcedureUsage.new('Mover')] }
> > +    it { expect(results.first[1]).to be_true }
>
> Ok, I think it is for now. But just I would like to notice that in te
> future, to be fully compatible with mumuki expectations, all expectations
> should have a binding they refer to.
>
> That means, you are not going to check if a procedure is used, but if a
> procedure is used within anothe
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uqbar-project/gobstones-spec/pull/9/files#r31285900>.
>